src/styles/blocks/catalog.scss

```
@@ -0,0 +1,25 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+
```

```suggestion
min-width: calc(200px * 4 + 48px * 3);
```
For this task you must set fixed width to have 4 columns in your grid (even on small screens).

* tip: use variables here